pkg/operator/starter.go

```diff
@@ -336,9 +336,12 @@ func prepareOauthOperator(controllerContext *controllercmd.ControllerContext, op
 	)
 
 	deploymentController := deployment.NewOAuthServerWorkloadController(
+		operatorCtx.operatorClient,
+		workloadcontroller.CountNodesFuncWrapper(operatorCtx.kubeInformersForNamespaces.InformersFor("").Core().V1().Nodes().Lister()),
+		workloadcontroller.EnsureAtMostOnePodPerNode,
 		operatorCtx.kubeClient,
+		operatorCtx.kubeInformersForNamespaces.InformersFor(metav1.NamespaceSystem).Core().V1().Nodes(),
```

Comment:
`Node` resources are cluster-wide.

Reply:
I _think_ this still works, it's also used here: https://github.com/openshift/cluster-authentication-operator/blob/ae01c18f5415feab7528792802dd66bdfbf49d6c/vendor/github.com/openshift/library-go/pkg/operator/apiserver/controllerset/apiservercontrollerset.go#L240

Reply:
on a second thought, I don't know _why_ it would work, I'll fix it both places